webapp/src/debuggerVariables.tsx

```diff
@@ -107,17 +117,37 @@ export class DebuggerVariables extends data.Component<DebuggerVariablesProps, De
             />);
         }
 
-        return <DebuggerTable header={variableTableHeader} placeholderText={placeholderText}>
-            {tableRows}
-        </DebuggerTable>
+        const previewVar = this.getVariableById(preview?.id);
+
+        return <div>
+            <DebuggerTable header={variableTableHeader} placeholderText={placeholderText}>
+                {tableRows}
+            </DebuggerTable>
+            { preview &&
+                <div id="debugger-preview"
+                    role="tooltip"
+                    className="debugger-preview"
+                    ref={this.handlePreviewRef}
+                    tabIndex={0}
+                    style={{
```

Comment:
I think an aria-label here saying something like `Current value for {variable}:` would be helpful

Reply:
Will do!